src/samples/p2p/peercall.html

```diff
@@ -96,11 +96,11 @@ <h2>LocalView</h2>
     </div>
     <div id="remote">
       <h2>RemoteView</h2>
-      <video width="320px" height="240px" id="remoteVideo" playsinline muted autoplay></video>
```

Comment:
Any reason to remove `muted`?

Reply:
if user can not heard audio , they will conside sdk issue